onebusaway-webapp/src/main/java/org/onebusaway/webapp/gwt/notification/NotificationWidgetPage.java

```diff
@@ -189,10 +193,11 @@ public Widget update(Context context) throws PageException {
 
     _tripId = context.getParam(PARAM_TRIP_ID);
     _stopId = context.getParam(PARAM_STOP_ID);
+    _serviceDate = context.getParam(PARAM_SERVICE_DATE);
 
-    System.out.println("tripId=" + _tripId + " stopId=" + _stopId);
+    System.out.println("tripId=" + _tripId + " stopId=" + _stopId + " serviceDate=" + _serviceDate);
 
-    if (_tripId == null || _stopId == null) {
+    if (_tripId == null || _stopId == null || _serviceDate == null) {
```

Comment:
I have a bug with Pop-up on arrival-and-departure-for-stop.action, if this parameter doesn't exist the pop-up don't showing.

Eg on Tampa: Without the parameter serviceDate: 

`http://tampa.onebusaway.org/where/standard/arrival-and-departure-for-stop.action?stopId=Hillsborough+Area+Regional+Transit_3165&tripId=Hillsborough+Area+Regional+Transit_119590&vehicleId=Hillsborough+Area+Regional+Transit_2217&stopSequence=70`

With the parameter serviceDate:

`http://tampa.onebusaway.org/where/standard/arrival-and-departure-for-stop.action?stopId=Hillsborough+Area+Regional+Transit_3165&tripId=Hillsborough+Area+Regional+Transit_119590&serviceDate=1452142800000&vehicleId=Hillsborough+Area+Regional+Transit_2217&stopSequence=70`

Then I added this validation.